src/jswrap_object.c

```diff
@@ -222,8 +222,20 @@ void jswrap_object_keys_or_property_names_cb(
 
     while (symbols) {
       unsigned int i;
-      for (i=0;i<symbols->symbolCount;i++) {
-        JsVar *name = jsvNewFromString(&symbols->symbolChars[symbols->symbols[i].strOffset]);
+      unsigned char symbolCount = READ_FLASH_UINT8(&symbols->symbolCount);
+      unsigned short strOffset = READ_FLASH_UINT16(&symbols->symbols[i].strOffset);
+      for (i=0;i<symbolCount;i++) {
+#ifndef USE_FLASH_MEMORY
+        JsVar *name = jsvNewFromString(&symbols->symbolChars[strOffset]);
```

Comment:
Yeah, it is used quite a bit (although quite a few of those will be with constant strings), but honestly I think in the scheme of things it's not actually going to make any noticeable difference to performance (like <1%).

I guess the other option is to have a 'flash-only' version of `jsvNewFromString` - I can just see this pattern of copy string to memory getting copied all over code as people try and get rid of strings from RAM, and the 64 byte buffer is going to cause trouble sooner or later when it's just copy+pasted.